app/services/report_template_format.rb

```diff
@@ -0,0 +1,45 @@
+class ReportTemplateFormat
```

Comment:
Aren't we actually implementing [Mime::Type](https://api.rubyonrails.org/classes/Mime/Type.html) here?

Reply:
I agree there's some overlap, but our class contains human names as we want to display them in UI, we're adding default and system logic. I'd say this is a layer on top and we could probably use Mime::Type instances instead of storing their strings, but until we use any feature of it, I'd not complicate it.

Reply:
I do not agree human names should be stored in code so deep, but it's usual practice in foreman, so I am good and thus with the explenation as well :)